### ISSUE TITLE
Top species assemblies do not contain that of subspecies

### ISSUE THREAD

**Issue by shenwei356** [pirovc/genome_updater#64]
Hi Vitor,

I'd like to keep top assemblies for each species using command (v0.5.0) below:

    time genome_updater.sh \
            -d "refseq" \
            -g "bacteria"\
            -c "reference genome,representative genome,na" \
            -l "complete genome,chromosome,scaffold,contig" \
            -f "genomic.fna.gz" \
            -A "species:10" \
            -o "output" \
            -t 20 \
            -m -a -p

However, when I check the filtered `assembly_summary.txt` file and note that assemblies of subspecies are not included. E.g.,
two *Akkermansia muciniphila*  ATCC BAA-835  (taxid 349741) assemblies of "Complete Genomes" are filtered out. Besides, only one assembly of "Complete Genome" are chosen out of  92, while some of "Scaffold" are chosen.

```
# -------------------------  Unfiltered --------------------------------
$ cut -f 1,5,6,8,9,12 assembly_summary.txt | grep 'Akkermansia muciniphila'
GCF_000020225.1 na      349741  Akkermansia muciniphila ATCC BAA-835    strain=ATCC BAA-835     Complete Genome
GCF_000723745.2 na      239935  Akkermansia muciniphila         Scaffold
GCF_001688765.2 na      239935  Akkermansia muciniphila strain=YL44     Complete Genome
GCF_002161325.1 na      239935  Akkermansia muciniphila strain=An78     Contig
GCF_002201495.1 na      239935  Akkermansia muciniphila strain=YL44     Chromosome
GCF_002884915.1 na      239935  Akkermansia muciniphila strain=GP24     Scaffold
GCF_002884975.1 na      239935  Akkermansia muciniphila strain=GP22     Conti
...

$ taxonkit list --ids 239935 -n -r 
239935 [species] Akkermansia muciniphila
  349741 [strain] Akkermansia muciniphila ATCC BAA-835

$ cut -f 1,5,6,8,9,12 assembly_summary_refseq.txt | grep 'Akkermansia muciniphila' | csvtk freq -Ht -f 2,3,6 -nr | csvtk pretty -Ht 
na                      239935   Complete Genome   92
na                      239935   Scaffold          68
na                      239935   Contig            52
na                      239935   Chromosome        3
na                      349741   Complete Genome   2
representative genome   239935   Complete Genome   1


# -------------------------  filtered --------------------------------
GCF_002201495.1 na      239935  Akkermansia muciniphila strain=YL44     Chromosome
GCF_004101765.1 na      239935  Akkermansia muciniphila strain=H2       Chromosome
GCF_008422615.1 na      239935  Akkermansia muciniphila strain=BIOML-A45        Scaffold
GCF_008422635.1 na      239935  Akkermansia muciniphila strain=BIOML-A44        Scaffold
GCF_008422665.1 na      239935  Akkermansia muciniphila strain=BIOML-A43        Scaffold
GCF_008422685.1 na      239935  Akkermansia muciniphila strain=BIOML-A39        Scaffold
GCF_008422695.1 na      239935  Akkermansia muciniphila strain=BIOML-A41        Scaffold
GCF_008422705.1 na      239935  Akkermansia muciniphila strain=BIOML-A40        Scaffold
GCF_009731575.1 representative genome   239935  Akkermansia muciniphila strain=JCM 30893        Complete Genome
GCF_016697425.1 na      239935  Akkermansia muciniphila strain=YL44     Chromosome

```

**Comment by shenwei356** [comment]
https://github.com/pirovc/genome_updater/blob/master/genome_updater.sh#L477

    col12["Complete genome"]=1;

Should be 

    col12["Complete Genome"]=1;

**Comment by pirovc** [comment]
Thanks for reporting, finding the solution and submitting the PR!

**Comment by shenwei356** [comment]
Glad to help. 

But I'm not sure whether limiting top assemblies of every taxid is also affected.

**Comment by pirovc** [comment]
It should, but I'll double check and add some test cases before releasing.

**Comment by pirovc** [comment]
Fixed in version 0.5.1 #69
